tests/rptest/services/action_injector.py

```diff
@@ -42,13 +52,20 @@ class DisruptiveAction:
     The action can be reversible, it also stores the set of affected nodes and the last node
     the action was applied on.
     """
+
+    is_reversible = False
+
     def __init__(self, redpanda: RedpandaService, config: ActionConfig,
                  admin: Admin):
         self.admin = admin
         self.config = config
         self.redpanda = redpanda
+        self.nodes_for_action = queue.PriorityQueue(
+            maxsize=self.redpanda.num_nodes)
+        now = int(time.time())
+        for node in self.redpanda.nodes:
+            self.nodes_for_action.put(NodePriority(now, node))
```

Comment:
Does this mean that there is no randomness in the order in which we choose nodes? Should we at least add them to the queue in the random order?

Reply:
This was an error on my part, I assumed that items with same priority will be returned in random order by the queue, but it is incorrect. Now I have added a utility to get random priority so that nodes are grouped according to the count of disruptive actions performed on them, and nodes with same count have the same range of priority. As the count increases the priority range also increases.